CHANGELOG.md

```diff
@@ -57,3 +57,20 @@ Other minor additions
   ```
 
 * Add version to library name
+
+* Add new properties to `Data.Vec.Properties`:
+  ```agda
+  take-distr-zipWith : ∀ {m n} → (f : A → B → C) →
+                       (u : Vec A (m + n)) → (v : Vec B (m + n)) →
+                       take m (zipWith f u v) ≡ zipWith f (take m u) (take m v)
```

Comment:
We usually remove the arguments in the CHANGELOG and just give the main thrust of the type, i.e. in this case we would just give it the type `take m (zipWith f u v) ≡ zipWith f (take m u) (take m v)`

Reply:
fixed.